code/lib/channel-websocket/src/index.ts

```diff
@@ -20,9 +20,9 @@ interface CreateChannelArgs {
 }
 
 export class WebsocketTransport {
-  private socket: WebSocket;
+  private socket?: WebSocket;
```

Comment:
I would prefer inlining the connect function in the constructor, to show TS that it is always defined. As it is private, it is really only used in the constructor anyway.